src/main/java/mil/dds/anet/AnetApplication.java

```diff
@@ -114,7 +120,12 @@ public void run(AnetConfiguration configuration, Environment environment) {
 		final DBIFactory factory = new DBIFactory();
 		final DBI jdbi = factory.build(environment, configuration.getDataSourceFactory(), "mssql");
 
-		logger.info("dictionary: {}", new JSONObject(configuration.getDictionary()).toString(2));
+		final JSONObject dictionary = getDictionary(configuration);
```

Comment:
This makes the dictionary structure being checked when doing `anet start anet.yml` . It should also happen during `anet check anet.yml` - possilby by changing/replacing the `CheckCommand.java` behavior